persist/fs/commitlog/commit_log.go

```diff
@@ -241,7 +244,7 @@ func (l *commitLog) write() {
 
 			continue
 		}
-
+		// TODO(prateek): track success/errors per namespace
```

Comment:
Meh, don't see the reason why we should?  It's all the one file, no reason why one would fail vs the other.